webapp/src/main/java/org/apache/atlas/web/security/AtlasKeycloakAuthenticationProvider.java

```diff
@@ -68,9 +80,27 @@ public Authentication authenticate(Authentication authentication) {
       }
     }
 
-    if(authentication.getName().startsWith("service-account-apikey")) {
+    if (authentication.getName().startsWith("service-account-apikey")) {
       // Increment the counter when the authentication is for a service account.
       Counter.builder("service_account_apikey_request_counter").register(MetricUtils.getMeterRegistry()).increment();
+
+      // Validate the token online with keycloak server if token introspection is enabled
+      LOG.info("isTokenIntrospectionEnabled: {}", isTokenIntrospectionEnabled);
```

Comment:
nit: we can avoid this log, since if `isTokenIntrospectionEnabled==true` then we already have a log within.